packages/template-retail-react-app/app/commerce-api/mock-data.js

```diff
@@ -2985,6 +2985,1625 @@ export const mockedCustomerProductListsDetails = {
     total: 1
 }
 
+// This wishlist has more varied products
+export const mockedCustomerProductListsDetails2 = {
```

Comment:
What happened to the generator function we were working on when we paired on this? Can you call out the specific bits that are different about this mock vs the other?

Reply:
I've moved this data into a separate file ("something.mock.js" format) that is in a more specific location than the "app/commerce-api/mock-data.js". Because the mocked data is intended to be used only for a particular test.

Reply:
@vmarta I think my main question got missed here. I realize I didn't really specify this, but I'm mainly interested in "how big is the difference between the existing mock data and this new mock data". If they're only 30% similar, I can understand not using the mock generator function pattern. If they're 90% similar, I would expect we want to use the generator pattern.

Reply:
@bfeister oh sorry, the new mocked data is completely different in this case.